### ISSUE TITLE
Export only correct answers

### ISSUE THREAD

**Issue by jmoeyersons** [dodona-edu/dodona#2561]
Feature request: is it possible to add an option so you can only export the last answer of students who had a correct answer?

**Comment by pdawyndt** [comment]
@jmoeyersons Be careful that this may have unwanted replications. This is actually what version 1.0 of Dodona was using, but in the very first evaluation that we took with Dodona, there was a student who at a certain point in time had a correct solution. However, two of the testcases were hardcoded in that solution. Afterwards, the student had made several attempts to get rid of the hardcoded stuff (knowing that we would penalise this during evaluation) but never succeeded in doing so. As a result, there was no way he could overrule the solution that we selected for assessment (which was the last correct solution before the deadline, or the last solution if none were correct). This is why we abandoned this heuristic, in favor of a heuristic (last solution before deadline, irrespective of its status) that gives full control to the student about which solution is handed in for evaluation.

If we decide to implement this export option, it should come with a clear warrant.
